docs/howto/import.md

```diff
@@ -151,19 +151,6 @@ the following policy needs to be attached to the lakeFS S3 service-account to al
 <div markdown="1" id="azure-storage">
 See [Azure deployment][deploy-azure-storage-account-creds] on limitations when using account credentials.
 
-### Azure Data Lake Gen2
-{:.no_toc}
-
-lakeFS requires a hint in the import source URL to understand that the provided storage account is ADLS Gen2
```

Comment:
We should document the change.  Otherwise someone who's already imported has no way of understanding what went wrong!

Reply:
Added deprecation message